.kokoro/generate-docs.sh

```diff
@@ -93,6 +95,12 @@ for bucket_item in $(gsutil ls 'gs://docs-staging-v2/docfx-python*' | sort -u -t
     ## upload docs
     # python3 -m docuploader upload docs/_build/html --metadata-file docs.metadata --staging-bucket "${STAGING_BUCKET}"
 
+    # Test running with the plugin version locally.
+    if [[ ${TEST_PLUGIN} == "true" ]]; then
```

Comment:
I think you need quotes around this in case it's not set?
```suggestion
    if [[ "${TEST_PLUGIN}" == "true" ]]; then
```